src/index/create.rs

```diff
@@ -129,9 +130,16 @@ impl IndexCreateStatement {
             primary: false,
             unique: false,
             index_type: None,
+            if_not_exists: false,
         }
     }
 
+    /// Create index if index not exists
```

Comment:
I think is better add doc tests instead `tests`. It will be like examples

Reply:
already add doc tests